<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NemezisV3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="asmTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="hextTextEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="convertBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert -&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="archsCombo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>x86_64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>arm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>aarch64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mips64</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="thumbBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumb</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bigEndianBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Big Endian</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertBackBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
